src/sentry/api/serializers/models/event.py

```diff
@@ -287,78 +304,41 @@ def serialize(self, obj, attrs, user):
         return result
 
 
-class SnubaEvent(object):
-    """
-        A simple wrapper class on a row (dict) returned from snuba representing
-        an event. Provides a class name to register a serializer against, and
-        Makes keys accessible as attributes.
-    """
-
-    # The list of columns that we should request from snuba to be able to fill
-    # out a proper event object.
-    selected_columns = [
-        'event_id',
-        'project_id',
-        'message',
-        'title',
-        'location',
-        'culprit',
-        'user_id',
-        'username',
-        'ip_address',
-        'email',
-        'timestamp',
-    ]
-
-    def __init__(self, kv):
-        assert len(set(self.selected_columns) - set(kv.keys())
-                   ) == 0, "SnubaEvents need all of the selected_columns"
-        self.__dict__ = kv
-
-
-@register(SnubaEvent)
-class SnubaEventSerializer(Serializer):
+class SimpleEventSerializer(EventSerializer):
     """
-        A bare-bones version of EventSerializer which uses snuba event rows as
-        the source data but attempts to produce a compatible (subset) of the
-        serialization returned by EventSerializer.
+    Simple event serializer that renders a basic outline of an event without
+    most interfaces/breadcrumbs. This can be used for basic event list queries
+    where we don't need the full detail. The side effect is that, if the
+    serialized events are actually SnubaEvents, we can render them without
+    needing to fetch the event bodies from nodestore.
     """
 
-    def get_tags_dict(self, obj):
-        keys = getattr(obj, 'tags.key', None)
-        values = getattr(obj, 'tags.value', None)
-        if keys and values and len(keys) == len(values):
-            results = []
-            for key, value in zip(keys, values):
-                key = key.split('sentry:', 1)[-1]
-                result = {'key': key, 'value': value}
-                query = convert_user_tag_to_query(key, value)
-                if query:
-                    result['query'] = query
-                results.append(result)
-            results.sort(key=lambda x: x['key'])
-            return results
-        return []
+    def get_attrs(self, item_list, user):
+        return {}
 
     def serialize(self, obj, attrs, user):
-        result = {
+        tags = [{
+            'key': key.split('sentry:', 1)[-1],
```

Comment:
Edge case, but this makes `mysentry:42` into `42`, that is not the same logic we use elsewhere.

Reply:
Not sure which logic is the best here, This is just a copy of what happens in `_get_tags_with_meta` above, but I agree that the matching is not very good. should I change it in both places? 

https://github.com/getsentry/sentry/pull/11071/files/73edf6796e7b9c774fbc748f521019e5e0a5701d#diff-60ceffc08be5480b5791bc0ef3458133L95

Reply:
Damn. I just realized we already did that. Yeah I guess so.